### DISCUSSION TITLE
API doesn't make a delivery

### DISCUSSION THREAD

**Discussion by kennethmaes** [OCA/odoo-community.org#188] [Q&A]
Hello everyone,

I have set up an API between WooCommerce and Odoo. So far so good, it links the correct product, prices, VAT rates, and such.

The API ensures that a sales order is created immediately (since the order has already been paid, so no quotation is needed).

However, it does not create a delivery for this order. If I create the same order manually, it does. It doesn't check the stock and doesn't fill in anything for the delivery status.

Or if I change to qty manual to 0, save it, and then change it back to 1, Odoo checks the stock. I think I miss something to trigger this action. 

Do I need to pass a specific value to the API? Does anyone know a solution to get this right?

Attached you find my sendfile to Odoo. 
[API Odoo.docx](https://github.com/user-attachments/files/17228397/API.Odoo.docx)

Best regards,

Kenneth Maes
[api odoo.json](https://github.com/user-attachments/files/17228399/api.odoo.json)

**Comment by pedrobaeza** [comment]
This seems something malicious with downloadable files that are very suspicious. Anyway, it's not referred to any OCA module.

**Comment by nilshamerlinck** [comment]
Probably not malicious but definitely lacking some context :sweat_smile:
 
Are you using an OCA module to build this API @kennethmaes?
